src/main/app/drafts/models/draftLegalClaim.ts

```diff
@@ -12,6 +12,7 @@ import { StatementOfTruth } from 'app/forms/models/statementOfTruth'
 import { Amount } from 'app/forms/models/amount'
 import { FeeAccount } from 'forms/models/feeAccount'
 import { DraftDocument } from '@hmcts/cmc-draft-store-middleware'
+import { PaymentResponse } from 'app/pay/model/paymentResponse'
```

Comment:
can you please check if `app/` is needed?

Reply:
does not have pay reference atm, so will not work